src/main/scala/com/schuwalow/todo/repository/TodoRepository.scala

```diff
@@ -9,29 +9,23 @@ import com.schuwalow.todo.{
   TodoItemPostForm
 }
 
-trait TodoRepository extends Serializable {
-
-  val todoRepository: TodoRepository.Service[Any]
-
-}
-
 object TodoRepository extends Serializable {
 
-  trait Service[R] extends Serializable {
+  trait Service extends Serializable {
 
-    def getAll(): ZIO[R, Nothing, List[TodoItem]]
+    def getAll(): ZIO[Any, Nothing, List[TodoItem]]
```

Comment:
These could be UIO

Reply:
Alright, I just had the impression that you are not using UIO, RIO etc. on purpose, because I didn't see it used anywhere... but I'm happy to refactor :-)

Reply:
If I did then it's something we should change :)